pages/common/shellcheck.md

```diff
@@ -0,0 +1,19 @@
+# shellcheck
+
+> Shell script static analysis tool.
+
+- Check a shell script:
+
+`shellcheck myscript.sh`
+
+- Override script's shebang (valid values: sh, bash, and ksh):
+
+`shellcheck --shell sh myscript.sh`
```

Comment:
You can provide multiple options using |. So `{{sh|bash|ksh}}`.

Reply:
Thanks. Addressed in new commit b97b8d8.